src/turbopuffer.test.ts

```diff
@@ -260,62 +260,58 @@ test("schema", async () => {
         full_text_search: {
           stemming: true,
           remove_stopwords: true,
-          case_sensitive: false
-        }
+          case_sensitive: false,
+        },
       },
       tags: {
         type: "[]string",
         full_text_search: {
           stemming: false,
           remove_stopwords: false,
-          case_sensitive: true
-        }
-      }
-    }
+          case_sensitive: true,
+        },
+      },
+    },
   });
 
   const schema = await ns.schema();
   expect(schema).toEqual({
     id: {
-      type: 'uint',
+      type: "uint",
       filterable: null,
-      bm25: null,
```

Comment:
we changed the rust side to skip serializing this if null

Reply:
yeah I've these removed in the non-node env support pr too!